src/functions/static_functions.rs

```diff
@@ -25,6 +25,17 @@ impl EasingFunction for Step {
 	}
 }
 
+/// Hold function, always returns A
```

Comment:
Maybe there could be two functions, one that always returns A and one that always returns B?

Reply:
IMO it's unnecessary since in order to achieve the same result you can simply delete frame A and set the easing of frame B to Hold

Reply:
Makes sense!